exercises/01-composite-ui/before/Divergent.Sales.API/Divergent.Sales.API.csproj

```diff
@@ -9,6 +9,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNet.WebApi.Cors" Version="5.2.3" />
     <PackageReference Include="Microsoft.AspNet.WebApi.OwinSelfHost" Version="5.2.4" />
+    <PackageReference Include="NServiceBus" Version="7.0.1" />
+    <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="2.1.0" />
+    <PackageReference Include="NServiceBus.Transport.Msmq" Version="1.0.0" />
+    <PackageReference Include="Radical.Bootstrapper.Windsor.AspNet" Version="1.0.0" />
```

Comment:
Exercise 1 should not have any NServiceBus references. The changes to this file should be reverted.